README.md

```diff
@@ -96,7 +96,7 @@ instructions used to create Azure IoT Edge Snap package. The walk-through is a g
 starting point for anyone trying to integrate Azure IoT Edge into their own snap
 package.
 
-API documentation can be found [here](http://azure.github.io/azure-iot-gateway-sdk/api_reference/c/html).
+API documentation can be found [here](http://azure.github.io/iot-edge/api_reference/c/html).
```

Comment:
This API doc link points specifically to the stuff for C. Can we link instead to the higher-level page (http://azure.github.io/iot-edge/)?